solana/programs/swap-layer/src/composite/mod.rs

```diff
@@ -164,6 +171,11 @@ pub struct ConsumeSwapLayerFill<'info> {
                 SwapLayerError::InvalidPeer,
             );
 
+            // msg!(
```

Comment:
Can we remove this?

Reply:
Removed here: https://github.com/wormholelabs-xyz/swap-layer/pull/15/commits/7e22d984f6f6b31d954e2eb64a80436bbff64501